packages/files-ui/src/Components/Modules/FileBrowsers/views/FilesList.tsx

```diff
@@ -505,6 +501,46 @@ const FilesList = ({ isShared = false }: Props) => {
     [selectedCids, selectedItems]
   )
 
+  // select item with SHIFT pressed
+  const handleSelectItemWithShift = useCallback(
+    (item: FileSystemItemType) => {
+      // item already selected
+      const isItemAlreadySelected = selectedItems.find((s) => s.cid === item.cid)
```

Comment:
You can use `selectedItems.~~contains~~includes(item.cid)` here.

Reply:
This implementation becomes a problem really, as selectedItems is an array of objects. simply checking the cid match works.

Reply:
sorry, I should have said `.includes` indead of contains. It's more elegant than `.find` in this case. 
```suggestion
      const isItemAlreadySelected = selectedItems.includes(item.cid)
```

and then you get a boolean, and you can check for `isItemAlreadySelected` instead of `isItemAlreadySelected.length`

Reply:
Yup, using the `includes` now.